pom.xml

```diff
@@ -119,16 +126,16 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
-                <version>3.0</version>
+                <version>3.5.1</version>
                 <configuration>
-                    <source>1.6</source>
-                    <target>1.6</target>
+                    <source>1.8</source>
+                    <target>1.8</target>
```

Comment:
Force pushed to use the maven.*.target properties. I kept the versions of the maven plugins in the pom: it's the only way to ensure a reproducible build.